WCSim.cc

```diff
@@ -59,27 +57,14 @@ int main(int argc,char** argv)
   // Set up the messenger hooks here, initialize the actual list after loading jobOptions.mac
   WCSimPhysicsListFactory *physFactory = new WCSimPhysicsListFactory();
 
-  // Currently, default model is set to BINARY
+  // Currently, default physics list is set to FTFP_BERT
```

Comment:
Add in a comment that we removed the custom physics list option in versions later than 1.6.0.